### PR TITLE
ES6 classes slides

### PR DESCRIPTION
**Pull request by nachosoloaga**

### CODE REVIEW

File: src/slides/class3/index.jsx [hunk 1]
```diff
@@ -12,5 +12,6 @@ export default () => (
     <Slides.Modules />
     <Slides.ArrowFunctions />
     <Slides.TemplateLiterals />
+    <Slides.Classes />
```
Comment (JulianPasquale) [inline src/slides/class3/index.jsx:15]: This should be before arrow functions, right?

File: src/slides/class3/slides/Classes.jsx [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+      <Heading margin="0px" fontSize="50px">
+        Expresiones de clases
+      </Heading>
+      <Text fontSize={30}>
+        También podemos definir clases utilizando{" "}
+        <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes#class_expressions">
+          expresiones de clases
+        </a>
+        .  Las expresiones de clase pueden ser nombradas o anónimas. El nombre
+        dado a la expresión de clase nombrada es local dentro del cuerpo de la
```
Comment (JulianPasquale) [inline src/slides/class3/slides/Classes.jsx:60]: This sentence is not so easy to read. I mean, I understand what you are saying but sound a bit complex

**Review by JulianPasquale — APPROVED** [review]

**Comment by JulianPasquale** [comment]
@nachosoloaga can we also explain what `this` means in a Class context?

File: src/slides/class3/index.jsx (hunk 1 above)
Comment (nachosoloaga) [reply src/slides/class3/index.jsx:15]: Yep, that's right 😅

File: src/slides/class3/slides/Classes.jsx (hunk 2 above)
Comment (nachosoloaga) [reply src/slides/class3/slides/Classes.jsx:60]: Yes, you're right. Maybe I can just replace it with `El nombre dado una clase mediante expresión de clase nombrada puede ser accedida mediante la propiedad name.`?

**Comment by nachosoloaga** [comment]
> @nachosoloaga can we also explain what `this` means in a Class context?

Yes! Sounds good, I'll add it.

File: src/slides/class3/slides/Classes.jsx (hunk 2 above)
Comment (JulianPasquale) [reply src/slides/class3/slides/Classes.jsx:60]: What about `El nombre asignado a una clase definida con esta notación se encuentra disponible al acceder a la propiedad name`?

Comment (nachosoloaga) [reply src/slides/class3/slides/Classes.jsx:60]: Yeah, I like that better

**Pull request merged**